projects.yml

```diff
@@ -5437,8 +5455,350 @@ salutem:
+  categories: [Encoding/Decoding]
+  platforms: [clj, cljs]
+
+nativity:
+  name: nativity
+  url: https://github.com/MnRA/nativity
+  categories: [Scripting]
+  platforms: [clj]
+
+headlessui-reagent:
+  name: headlessui-reagent
+  url: https://github.com/mainej/headlessui-reagent
+  categories: [React Interfaces]
+  platforms: [clj]
+
+trikl:
+  name: Trikl
+  url: https://github.com/lambdaisland/trikl
+  categories: [GUIs]
+  platforms: [clj]
+
+edamame:
+  name: Edamame
+  url: https://github.com/borkdude/edamame
+  categories: [EDN Parsers]
+  platforms: [clj, cljs]
+
+pure-frame:
+  name: pure-frame
+  url: https://github.com/binaryage/pure-frame
+  categories: [React Frameworks]
+  platforms: [cljs]
+
+freerange:
+  name: freerange
+  url: https://github.com/nextjournal/freerange
+  categories: [React Frameworks]
+  platforms: [cljs]
+
+liquid:
+  name: liquid
+  url: https://github.com/mogenslund/liquid
+  categories: [Editor]
+  platforms: [clj]
+
+nextjournal-clojure-mode:
+  name: clojure-mode
+  url: https://github.com/nextjournal/clojure-mode
+  categories: [Editor]
+  platforms: [cljs]
+
+reclur:
+  name: reclur
+  url: https://github.com/kjkostlan/reclur
+  categories: [Editor]
+  platforms: [clj]
+
+calcit-editor:
+  name: Calcit Editor
+  url: https://github.com/Cirru/calcit-editor
+  categories: [Editor]
+  platforms: [clj, cljs]
+
+clooj:
+  name: clooj
+  url: https://github.com/arthuredelstein/clooj
+  categories: [Editor]
+  platforms: [clj]
+
+betterCode:
+  name: BetterCode
+  url: https://github.com/vision-05/betterCode
+  categories: [Editor]
+  platforms: [clj]
+
+skija:
+  name: Skija
+  url: https://github.com/HumbleUI/Skija
+  categories: [Graphics, GUIs]
+  platforms: [clj]
+
+lezer-clojure:
+  name: lezer-clojure
+  url: https://github.com/lezer-parser/clojure
+  categories: [EDN Parsers]
+  platforms: [clj]
+
+org-parser:
+  name: org-parser
+  url: https://github.com/200ok-ch/org-parser
+  categories: [Org Parsers]
+  platforms: [clj, cljs]
+
+debux:
+  name: debux
+  url: https://github.com/philoskim/debux
+  categories: [Debugging]
+  platforms: [clj, cljs]
+
+clj-watson:
+  name: clj-watson
+  url: https://github.com/clj-holmes/clj-watson
+  categories: [Vulnerability Checking]
+  platforms: [clj]
+
+workspaces:
+  name: workspaces
+  url: https://github.com/nubank/workspaces
+  categories: [ClojureScript Development]
+  platforms: [cljs]
+
+grape:
+  name: Grape
+  url: https://github.com/bfontaine/grape
+  categories: [EDN, Parsing]
+  platforms: [clj]
+
+grasp:
+  name: grasp
+  url: https://github.com/borkdude/grasp
+  categories: [EDN, Parsing]
+  platforms: [clj]
+
+babashka-fs:
+  name: babashka.fs
+  url: https://github.com/babashka/fs
+  categories: [Scripting, Filesystem Utilities]
+  platforms: [clj]
+
 truegrit:
   name: True Grit
   url: https://github.com/KingMob/TrueGrit
   categories: [Retrying Failures]
-  platforms: [clj]
+  platforms: [clj]
```

Comment:
Line ending erroneously removed.

Reply:
I don't understand from looking at this diff. Is the problem that there should be a newline at the end of the file but there is no newline?